doctrine.rst

```diff
@@ -569,26 +571,18 @@ a controller, this is pretty easy. Add the following method to the
         return new Response('Saved new product with id '.$product->getId());
     }
 
-    // you can also receive the $em as an argument
-    public function editAction(EntityManagerInterface $em)
+    // you can use the registry to fetch other em's, if you have mutiple
+    public function editAction(ManagerRegistry $registry)
```

Comment:
A question newcomers (and not only them) may ask: why sometimes I need to inject an interface (`EntityManagerInterface`) and sometimes just a class (`ManagerRegistry`).